src/components /EditSighting.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect, startTransition } from "react";
+import axios from "axios";
+import { serverURL } from "../ServerURL";
+import { useNavigate, Link, useParams } from "react-router-dom";
+import { Card } from "antd";
+
+const EditSighting = () => {
+  const [newSighting, setNewSighting] = useState({
+    date: "",
+    location: "",
+    notes: "",
+  });
+  const [sightingID, setSightingID] = useState(0);
+  const [originalSighting, setOriginalSighting] = useState({});
```

Comment:
in case original here stands for "it exists in the database", then we usually could use the word "persistence". "persistenceSighting"